examples/showcase/link-traversal/index.html

```diff
@@ -23,7 +23,7 @@
       </a-assets>
       <a-link href="city.html" position="-3.5 1.5 -1.0" image="#thumbCity"></a-link>
       <a-link href="sunrise.html" position="0 1.5 -1.0" image="#thumbSunrise"></a-link>
-      <a-link href="mountains.html" position="3.5 1.5 -1.0" image="#thumbMountains"></a-link>
+      <a-link href="mountains.html" position="3.5 2.0 -1.0" image="#thumbMountains" scale="2 2 2" preview-distance="30"></a-link>
```

Comment:
Can we leave example as it was?

Reply:
Sure.  It's just hard to understand the need for this, without a larger-scaled link.

Reply:
I've removed the changes.  You can transiently set the scale of a link to see the effects now.